services/Directory/data/Language.json

```diff
@@ -898,5 +898,10 @@
     "id": 180,
     "iso6391": "zu",
     "name": "Zulu"
+  },
+  {
+    "id": 181,
+    "iso6391": "pap",
```

Comment:
The latest build error is complaining that `pap` is three characters when the database only expects two.

It looks like `pap` isn't an ISO 639-1 code, but rather an ISO 639-2 code. I can't seem to find if it has an ISO 639-1 code...

I'm not sure how to move forward... I'm open to changing what code standardization we use on FilterLists if that's better, but that would take some work...

Reply:
I opened #3502 

For now, I would suggest just not adding this language. It looks like it is only being applied to a single list here. Once we figure out a fix for #3502 , we can certainly add the language to that list.